pkgs/development/libraries/qt-5/5.4/default.nix

```diff
@@ -152,7 +152,7 @@ let
 
       multimedia = callPackage
         (
-          { qtSubmodule, base, declarative
+          { qtSubmodule, base, declarative, pkgconfig
```

Comment:
I'm not so sure that this is only relevant for qutebrowser though.  This issue potentially affects every program depending on QtMultimedia or QtWebKit as otherwise video support in Qt is broken.

I guess the cleanest solution from a closure-size point of view would be to split the multimedia module into multimedia containing only the libraries and headers, and multimedia-plugins containing just the plugins.  As long as QT_PLUGIN_PATH points to the plugins, everything should work just fine.  This could probably be done with multiple outputs, but I don't know how to integrate that correctly with the qt setupHook.  I didn't split the packages up because enabling gstreamer support was probably intended (the dependencies were all there), it was just pkgconfig that was missing.

What is the general plan for Qt plugins?  Are they supposed to be installed system-wide?  QT_PLUGIN_PATH contains system and users profiles here, then there is a config.qtPlugins option setting QT_PLUGIN_PATH as well, and then there is the qt setupHook setting it as well.  Are they supposed to be set by program wrappers?

cc @bbenoist @qknight @ttuegel